jsonschemas/ArrayOfPrimitives.jsonschema

```diff
@@ -1,94 +1,101 @@
 {
     "$schema": "http://json-schema.org/draft-04/schema#",
-    "properties": {
-        "description": {
-            "oneOf": [
-                {
-                    "type": "null"
+    "$ref": "#/definitions/ArrayOfPrimitives",
+    "id": "ArrayOfPrimitives",
```

Comment:
I'm seeing the "id" value repeated twice per schema, which doesn't make it a valid ID. As mentioned in the other comment, I would remove the top-level id as we're purely using $ref for defining top-level properties.

Reply:
I've removed ID from the whole lot (top-level plus all definitions). Hopefully this is acceptable for your use case.

Reply:
Thanks.

To be clear I am fine with IDs, but they must be valid to the draft-4 JSON schema specification if we should include them (for anyone's use case). The spec requires a scheme for the ID for example.

It's a little easier to add fields than remove fields.

https://datatracker.ietf.org/doc/html/rfc3986#section-1.1.1
https://datatracker.ietf.org/doc/html/draft-zyp-json-schema-04#section-7.2.1